modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -241,6 +241,46 @@ CV__DNN_INLINE_NS_BEGIN
 
     };
 
+    class CV_EXPORTS EinsumLayer : public Layer
+    {
+    public:
+        static Ptr<EinsumLayer> create(const LayerParams& params);
+    /** @brief This function performs array summation based
```

Comment:
I think the docstring should be before `class CV_EXPORTS EinsumLayer : public Layer`

Reply:
I copied it from LSTM layer. [Here](https://github.com/Abdurrahheem/opencv/blob/86657ed9e697f28d395f8def6b6a7a62941da6c9/modules/dnn/include/opencv2/dnn/all_layers.hpp#L96)

Reply:
check https://pullrequest.opencv.org/buildbot/export/pr/24037/docs/dd/d15/classcv_1_1dnn_1_1EinsumLayer.html

docstring should lead class or method definition, like here: https://pullrequest.opencv.org/buildbot/export/pr/24037/docs/d2/dce/classcv_1_1dnn_1_1GRULayer.html

Reply:
Done!